src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/Metrics/IomtMetrics.cs

```diff
@@ -175,6 +176,23 @@ public static Metric NotSupported()
             return _notSupported;
         }
 
+        /// <summary>
+        /// A metric for when a FHIR resource has been saved.
+        /// </summary>
+        /// <param name="resourceType">The type of FHIR resource that was saved.</param>
+        /// <param name="resourceOperation">The operation performed on the FHIR resource.</param>
+        public static Metric FhirResourceSaved(ResourceType resourceType, ResourceOperation resourceOperation)
+        {
+            return new Metric(
+                $"FhirResourceSaved",
+                new Dictionary<string, object>
+                {
+                    { _nameDimension, $"{resourceType}FhirResource{resourceOperation}" },
```

Comment:
So this will result in ObservationFhirResourceCreated & ObservationFhirResourceUpdated?  Right now we are only using this in exceptions so we have exceptions like DeviceNotFoundException, PatientNotFoundException, DeviceIdentityNotDefinedException, etc.  It is currently implicit that these are FHIR Resources.  Since we are capturing the operation dimension as FHIRConversion the FHIR Resource aspect seems implicit.  Thoughts on removing for succinctness and alignment with other telemetry?

Reply:
Agree it makes sense to align. Updated. It will now result in: ObservationCreated & ObservationUpdated